### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by jaredinyaagha1**
A working navbar, complete with a title and buttons that increase the overall functionality of the application

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,14 +9,44 @@
...
+      <span data-responsive-toggle="topbar-responsive" data-hide-for="medium">
+        <button class="menu-icon" type="button" data-toggle></button>
+      </span>
+      <a class="topbar-responsive-logo" href="#"><strong>Drink Finder - Home</strong></a>
+    </div>
+    <div id="topbar-responsive" class="topbar-responsive-links">
+      <div class="top-bar-right">
+        <ul class="menu simple vertical medium-horizontal">
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About</a></li>
```
Comment (brOmarito) [inline index.html:30]: Let's remove the About and News options since we're not going to have those changes

File: index.html [hunk 2]
```diff
@@ -9,14 +9,44 @@
...
+          <li><a href="#">News</a></li>
+          <li><a href="#">Contact</a></li>
```
Comment (brOmarito) [inline index.html:32]: Let's comment out Contact for now unless we have time to add a contacts page

File: index.html [hunk 3]
```diff
@@ -9,14 +9,44 @@
...
+    <div class="top-bar-title">
+      <span data-responsive-toggle="topbar-responsive" data-hide-for="medium">
+        <button class="menu-icon" type="button" data-toggle></button>
+      </span>
+      <a class="topbar-responsive-logo" href="#"><strong>Drink Finder - Home</strong></a>
+    </div>
+    <div id="topbar-responsive" class="topbar-responsive-links">
+      <div class="top-bar-right">
+        <ul class="menu simple vertical medium-horizontal">
+          <li><a href="#">Home</a></li>
```
Comment (brOmarito) [inline index.html:29]: For home, can you add the redirect to the home page? href would be something like "index.html"

File: index.html [hunk 4]
```diff
@@ -9,14 +9,44 @@
...
       <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/foundation-sites@6.6.3/dist/css/foundation.min.css" integrity="sha256-ogmFxjqiTMnZhxCqVmcqTvjfe1Y/ec4WaRj/aQPvn+I=" crossorigin="anonymous">
       <link rel="stylesheet" href="./assets/style.css">
   </head>
-  <body>
 
-    <header>
-      <h1>Drink Finder</h1>
-    </header>
+<body>
+  <nav class="top-bar topbar-responsive">
```
Comment (brOmarito) [inline index.html:19]: Can you also add this to the results page?

**Review by brOmarito — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (brOmarito) [reply index.html:19]: Never mind, we can do this in a separate PR

File: index.html (hunk 1 above)
Comment (jaredinyaagha1) [reply index.html:30]: Done

File: index.html (hunk 2 above)
Comment (jaredinyaagha1) [reply index.html:32]: Done

File: index.html (hunk 3 above)
Comment (jaredinyaagha1) [reply index.html:29]: Done

**Pull request merged**